operators/src/machine_learning/onnx.rs

```diff
@@ -251,7 +251,8 @@ where
 
                 // extract the values as a raw vector because we expect one prediction per pixel.
                 // this works for 1d tensors as well as 2d tensors with a single column
-                let predictions = predictions.into_owned().into_raw_vec();
+                let (predictions, offset) = predictions.into_owned().into_raw_vec_and_offset();
+                assert!(offset.is_none() || offset == Some(0));
```

Comment:
debug_assert, or throw an error instead of crashin the engine here?

Reply:
[fce8876](https://github.com/geo-engine/geoengine/pull/994/commits/fce8876e48623769438ddc4d946e6fa57ecabc9f)